### ISSUE TITLE
Hand cursor in scenario editor's invention list has wrong hotspot

### ISSUE THREAD

**Issue by e-foley** [OpenRCT2/OpenRCT2#2394]
In OpenRCT2 build 1487, when you're rearranging the rides in your scenario editor invention list, the large hand cursor highlights items at the tip of the fingers (top of the icon).  The original RCT2 highlights items at the hand's center, which makes more sense because the intent is to "grab" the rides and move them between sections.

Unfortunately, I can't take screenshot of the issue because doing so hides the cursor.  Please try it for yourself.

**Comment by duncanspumpkin** [comment]
I tried a variety of different positions and I felt that the top made the most sense. Not sure what other people feel would be best.

**Comment by HaasJona** [comment]
I agree with @e-foley that the current cursor position feels awkward. I think the hotspot should be between the thumb and the index finger. Or just use a better (os provided) cursor anyway, this hand looks weird anyway. For example Linux provides the following cursor: 
![move](https://cloud.githubusercontent.com/assets/4310707/11456670/6ebfc116-9690-11e5-9333-7930f7e62ce5.png)

**Comment by e-foley** [comment]
I kind of like the original's placement,  though I wouldn't be opposed to a smaller version of the logo: the original sticks out.

**Comment by 733737** [comment]
is this still an issue
